garglk/garglk.ini

```diff
@@ -42,7 +42,13 @@ caps          0               # Force uppercase input  -- 0=off 1=on
 graphics      1               # enable graphics
 sound         1               # enable sound
 
-fullscreen    0               # set to 1 for fullscreen (Windows/Unix only)
+fullscreen    0               # set to 1 for fullscreen
+
+hires         1               # set to 1 to enable Retina display (macOS) / HiDPI (Windows)
+zoom          1.0             # set display zoom
```

Comment:
These don't look to actually be configurable, i.e. there's no code in `config.cpp` to set them from the config file.  The hard-coded values are always used.